docs/layouts/shortcodes/generated/orc_configuration.html

```diff
@@ -40,10 +40,16 @@
         </tr>
         <tr>
             <td><h5>orc.compression.zstd.level</h5></td>
-            <td style="word-wrap: break-word;">1</td>
+            <td style="word-wrap: break-word;">3</td>
             <td>Integer</td>
             <td>Define the compression level to use with ZStandard codec while writing data. The valid range is 1~22.</td>
         </tr>
+        <tr>
+            <td><h5>orc.compression.zstd.windowlog</h5></td>
```

Comment:
See OrcOptions, there is no orc.compression.zstd.windowlog.

Reply:
> See OrcOptions, there is no orc.compression.zstd.windowlog.

Paimon OrcOptions is useless at the moment.
according to org.apache.paimon.format.orc.OrcWriterFactory#getWriterOptions
org.apache.orc.OrcFile.WriterOptions convert form writerProperties direct.
so orc.compression.zstd.windowlog is support by OrcFile.WriterOptions.

Reply:
`OrcOptions` is used to generate documentation.